### ISSUE TITLE
Message "C:\Program Files\nodejs\\node.exe: Files/Microsoft is not supported in NODE_OPTIONS" when launching debug config

### ISSUE THREAD

**Issue by burdeasa** [microsoft/vscode-js-debug#569]
Since installing VSCode 1.47.0, whenever I attempt to debug, I see the following in the debug console:

```
C:\Program Files\nodejs\\node.exe: Files/Microsoft is not supported in NODE_OPTIONS
C:\Program Files\nodejs\\node.exe: Files/Microsoft is not supported in NODE_OPTIONS
Process exited with code 9
```

I have VSCode installed at the system path (at C:\Program Files\Microsoft VS Code) and I am using node.js v8.12.0 (yeah, I know it's old). So, I suspect there is an issue with the handling spaces in the path to the node boot loader.

Here is my launch config:
```
        {
            "name": "Debugger",
            "type": "node",
            "request": "launch",
            "runtimeExecutable": "${workspaceFolder}/launchElectron",
            "windows": {
                "runtimeExecutable": "${workspaceFolder}/launchElectron.cmd",
            },
            "runtimeArgs": [
                "start",
                "chromeDebug",
            ],
            "timeout": 90000,
            "program": "${workspaceFolder}/main.js",
            "protocol": "legacy",
            "port": 5858
        },

```

**Comment by connor4312** [comment]
Thanks for the issue! Please see https://github.com/microsoft/vscode/issues/102035#issuecomment-656384728

**Comment by connor4312** [comment]
A fix for this issue has now been released in VS Code 1.47.2. If you previously had to opt-out of the new debugger, you should be safe to turn it back on. Thanks again for the report!
